Docs/source/Software/Tasks.rst

```diff
@@ -2,6 +2,36 @@
 Real Time Operating System Tasks
 ********************************
 
+Initialization Task: Alex Koo
+===============================
+
+Purpose
+    Create tasks by calling "OSTaskCreate()" and provide argument specifying to RTOS how the task will be managed.
+    The order of the semaphore Initialization matters because if the fault semaphore isn't initialized first and a fault is called we do not go into the fault state.  
+
+
+Functionality
+    1) The TCB, Task name, Task function argument, Priority, Stack, Watermark limit for debugging, Stack size, 
+    Queue size, Time quanta, Extension pointer, Options, Return err code is provided. 
+
+    2) Defines the priority level for the tasks
+    
+    3) Specifies the size of the task's stack in number of elements which corresponds to the number of bytes available
+    on the stack 
+
+Priority
+    The initialization has the highest priority. 
+
+Timing Requirements
+    Initialization Task also creates the semaphores which is a programming construct that helps us achieve concurrency 
+    by implementing both synchronization and mutual exclusion. :Semaphores`<Semaphore>`are used when a task wants exclusive 
+    access to a resource, needs to synchronize its activities with an ISR or a task, or is waiting until an event occurs.
+    The task deletes itself after running the time requirements and saves the contents of floating point registers whenever there is a context switch. 
+
+Yields
+   There is no yield.
+
+
```

Comment:
Go into the docs folder and run 'make html' every time you make a change and see if it builds properly. If you see any warnings for things you didn't write then you don't have to change it.